packages/kbn-securitysolution-exceptions-common/scripts/openapi_bundle.js

```diff
@@ -28,6 +28,11 @@ const ROOT = resolve(__dirname, '..');
             "Exceptions API allows you to manage detection rule exceptions to prevent a rule from generating an alert from incoming events even when the rule's other criteria are met.",
         },
       },
+      assignTag: {
+        name: 'Security Solution Exceptions API',
+        description:
+          "Exceptions API allows you to manage detection rule exceptions to prevent a rule from generating an alert from incoming events even when the rule's other criteria are met.",
```

Comment:
With the addition of `assignTag`, we now have two places where the final produced document can be modified. For `info`, `servers`, `security`, and `components`, we have to use `prototypeDocument`, but for `tags`, we need to use `assignTag`. This makes the bundler API a bit more convoluted. Can we use a single place for tasks like overwriting or extending the resulting OpenAPI document?

Reply:
Technically it's possible to extend `prototypeDocument` with `tags` and allow to specify only a single tag. The problem here is that `assignTag` is accepted only by `bundle()` utility while `merge()` intentionally doesn't accept it. It may introduce confusion. Ideally it requires having to `PrototypeDocument` types one for `bundle()` and the other one for `merge()`.